model/post.go

```diff
@@ -62,7 +62,8 @@ const (
 	POST_PROPS_OVERRIDE_ICON_URL   = "override_icon_url"
 	POST_PROPS_OVERRIDE_ICON_EMOJI = "override_icon_emoji"
 
-	POST_PROPS_MENTION_HIGHLIGHT_DISABLED = "mentionHighlightDisabled"
+	POST_PROPS_MENTION_HIGHLIGHT_DISABLED = "disable_channel_highlight"
```

Comment:
As per my comment on the other PR, this should be reverted to camel case since even though it's inconsistent, changing this breaks compatibility.

Reply:
Done! Thanks